website/pages/use-cases/simple-container-orchestration.jsx

```diff
@@ -2,8 +2,30 @@ import UseCasesLayout from 'components/use-case-page'
 import TextSplitWithCode from '@hashicorp/react-text-split-with-code'
 import TextSplitWithImage from '@hashicorp/react-text-split-with-image'
 import FeaturedSliderSection from 'components/featured-slider-section'
+// Imports below are used in getStaticProps only
+import highlightData from '@hashicorp/nextjs-scripts/prism/highlight-data'
 
-export default function SimpleContainerOrchestrationPage() {
+export async function getStaticProps() {
+  const codeBlocksRaw = {
+    containerOrchestration: {
+      code:
+        'task "webservice" {\n  driver = "docker"\n\n  config {\n    image = "redis:3.2"\n    labels {\n      group = "webservice-cache"\n    }\n  }\n}',
+      language: 'hcl',
+    },
+    windowsSupport: {
```

Comment:
This `windowsSupport` snippet doesn't strictly need to be in `getStaticProps` since it doesn't require highlighting - I've just added it for consistency in how code block feature sections are managed.